modules/specialized-guides/pages/salt/salt-terminology.adoc

```diff
@@ -59,6 +58,7 @@ This is a difference in terminology only.
 
 Modules::
 Functions within Salt are stored in modules.
+Salt modules are stored on clients and the {productname} Server within the [path]``/usr/lib/python*/site-packages/salt/`` directory.
 There are many types of Salt modules, including state and execution modules.
 For a complete list of available Salt modules, see https://docs.saltstack.com/en/latest/ref/index.html.
 Alternatively, you can write your own Salt modules using Python.
```

Comment:
Drive-by edit:

I'd switch the sequence of these sentences and put the "more info" link in a para of its own:
```suggestion
You can write your own Salt modules using Python.
+
For a complete list of available Salt modules, see https://docs.saltstack.com/en/latest/ref/index.html.
```